web/book/src/tutorial/aggregation.md

```diff
@@ -1,95 +1,98 @@
 # Aggregation
 
 A key feature of analytics is reducing many values down to some summary. This
-act is called aggregation and always includes an aggregation function, for
-example `average` or `sum`:
+act is called "aggregation" and always includes a function &mdash; for
+example, `average` or `sum` &mdash; that reduces the table to a single row.
 
-<!-- aggregating ids does not make much sense, should we switch to some other table? -->
+**aggregate:** The `aggregate` transform takes a tuple to create one or many new columns
```

Comment:
Could we replace these inline emphases with more minor headings? Or you like the current style?